setup.py

```diff
@@ -325,13 +325,14 @@ def run(self):
     ext_modules=ext_modules,
     cmdclass=cmdclass,
     packages=packages,
+    license='MIT',
```

Comment:
We're moving to Apache 2.0 licensing with the shift to LFAI (perhaps in the future, if not now?). @prasanthpul might have more thoughts on this.

Reply:
we'll need to update once we actually switch (maybe later this month). at this moment it MIT